carsus/io/tests/test_cmfgen.py

```diff
@@ -54,6 +74,29 @@ def si2_pho_fname(refdata_path):
 def coiv_pho_fname(refdata_path):
     return os.path.join(refdata_path, 'cmfgen', 'photoionization_cross_sections', 'phot_data_gs')
 
+@with_refdata
+@pytest.fixture()
+def si1_data_dict(si2_osc_kurucz_fname):
+    si1_levels = CMFGENEnergyLevelsParser(si2_osc_kurucz_fname)  #  (carsus) Si 1 == Si II
+    si1_lines = CMFGENOscillatorStrengthsParser(si2_osc_kurucz_fname)
+    return {'Si 1': dict(levels = si1_levels, lines = si1_lines)}
+
+@with_refdata
+@pytest.fixture()
+def si1_reader(si1_data_dict):
+    return CMFGENReader(si1_data_dict)
+
+@with_refdata
+@pytest.fixture()
+def si2_levels_head_df():
+    return pd.read_csv(StringIO(si2_levels_head), delim_whitespace=True, names=['energy', 'j', 'label', 'method', 'priority'])
```

Comment:
I would reformat this.

Reply:
- Yes, requires reference data merged.
- Yes, we talked about using `black` in Carsus in the near future.